tests/js/spec/components/idBadge/teamBadge.spec.tsx

```diff
@@ -34,9 +30,7 @@ describe('TeamBadge', function () {
 
   it('updates state from props', async function () {
     const team = TestStubs.Team();
-    const {rerender} = mountWithTheme(<TeamBadge team={team} />, {
-      context: TestStubs.routerContext(),
-    });
+    const {rerender} = mountWithTheme(<TeamBadge team={team} />);
 
     rerender(<TeamBadge team={TestStubs.Team({slug: 'new-team-slug'})} />);
```

Comment:
This test here works now because TestStubs.Team creates a new team object which fires the useEffect - we should instead mutate the team before this rerender call to mimic the store implementation

```
team.avatar = "new avatar"
rerender(<TeamBadge team={team} />);
```